combat-trainer.lic

```diff
@@ -715,6 +715,8 @@ class SpellProcess
       end
     end
 
+    waitrt? while XMLData.prepared_spell == 'Vivisection' && game_state.use_stealth_attack? && ! hide?
```

Comment:
I'd use the xml over DRSpells, we made those DRSpells entries because we didn't know the game was tracking the data for us.